rs/execution_environment/src/execution/call_or_task/tests.rs

```diff
@@ -607,20 +607,18 @@ fn hitting_page_delta_limit_fails_message_system_subnet() {
     );
 }
 
-fn hitting_page_delta_limit_fails_for_install_code_helper(with_native_stable_mem: bool) {
+#[test]
+fn hitting_page_delta_limit_fails_for_install_code() {
     let no_pages_upgrade = 10;
     // A large enough limit that will never be triggered.
     let no_pages_other_messages = no_pages_upgrade * 10_000_000;
-    let mut test =
-        ExecutionTestBuilder::new().with_stable_memory_dirty_page_limit(StableMemoryPageLimit {
+    let mut test = ExecutionTestBuilder::new()
+        .with_stable_memory_dirty_page_limit(StableMemoryPageLimit {
             upgrade: NumOsPages::new(no_pages_upgrade),
             message: NumOsPages::new(no_pages_other_messages),
             query: NumOsPages::new(no_pages_other_messages),
-        });
-    if with_native_stable_mem {
-        test = test.with_non_native_stable();
-    }
```

Comment:
This appears to have been a benign bug. We actually do the reverse of what the flag means, which also means that the assert we should keep is the reverse one (see a bit lower in the code).

Reply:
nice catch, thanks!